ui/src/components/dataset/ingestion/IngestionInfo.vue

```diff
@@ -51,12 +50,25 @@
             <tr>
               <td>Project</td>
               <td class="metadata">
-                <router-link
-                  :to="`/projects/${props.project?.id}`"
-                  target="_blank"
+                <div v-if="props.project">
+                  <router-link
+                    :to="`/projects/${props.project.id}`"
+                    target="_blank"
+                  >
+                    {{ props.project.name }}
+                  </router-link>
+                </div>
+                <OutlinedAlert
+                  v-else-if="props.creatingNewProject"
+                  color="secondary"
+                  border="left"
+                  size="medium"
+                  padding-direction="left"
+                  padding-amount="sm"
+                  icon="info"
```

Comment:
Show user an alert telling them that a new Project will be created for this Dataset-creation action.